### ISSUE TITLE
pip install fails on debian stretch w/ python version 2.7.13

### ISSUE THREAD

**Issue by brucesm** [mohd-akram/base16-pygments#3]
See title, error output is below. Fixing is trivial, also see below.

```
root@orangepizeroplus2:~# pip install pygments-base16
Collecting pygments-base16
  Using cached pygments-base16-1.0.2.tar.gz
    Complete output from command python setup.py egg_info:
 setup.py+                                                                                                          buffers 
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-Hx3GC5/pygments-base16/setup.py", line 8, in <module>
        with open(path.join(base_dir, 'README.rst'), encoding='utf-8') as f:
    TypeError: 'encoding' is an invalid keyword argument for this function
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-Hx3GC5/pygments-base16/
```

```
diff --git a/setup.py b/setup.py
index 43c6621..f18ecac 100644
--- a/setup.py
+++ b/setup.py
@@ -1,4 +1,5 @@
 from os import path
+from io import open
 from glob import glob
 from string import capwords
 from setuptools import setup
```

**Comment by mohd-akram** [comment]
Can you use Python 3?

`pip3 install pygments-base16`

I can fix it since it's only one line, but it's better to just use Python 3.

**Comment by mohd-akram** [comment]
Fixed in e58bc0f3dee460ccd1e9cd7954c03637dd53a837.
